src/main/scala/org/broadinstitute/dsde/firecloud/service/FireCloudDirectives.scala

```diff
@@ -24,31 +23,18 @@ object FireCloudDirectiveUtils {
 trait FireCloudDirectives extends spray.routing.Directives with PerRequestCreator with spray.httpx.RequestBuilding {
   def respondWithJSON = respondWithMediaType(`application/json`)
 
-  def passthrough(unencodedPath: String, methods: HttpMethod*): Route =
-    passthrough(Uri(unencodedPath), methods:_*)
-
-  def passthrough(uri: Uri, methods: HttpMethod*): Route =
-    passthrough(requestCompression = true, uri, methods:_*)
+  def passthrough(unencodedPath: String, methods: HttpMethod*): Route = {
+    passthrough(Uri(unencodedPath), methods: _*)
+  }
 
-  def passthrough(requestCompression: Boolean, unencodedPath: String, methods: HttpMethod*): Route =
-    passthrough(requestCompression, Uri(unencodedPath), methods:_*)
+  def passthrough(requestCompression: Boolean, unencodedPath: String, methods: HttpMethod*): Route = {
+    passthrough(Uri(unencodedPath), methods: _*)
+  }
```

Comment:
this version of `passthrough` doesn't make sense anymore. It accepts a `requestCompression` argument and then immediately discards it. From a quick search, it's only called from 4 places, all in `EntityService`. Could you update those 4 calls and remove this signature?

Reply:
Great! The same seemed to be the case for `generateExternalHttpPerRequestForMethod` and `externalHttpPerRequest`, so I cleaned up those as well.